### PR TITLE
Python test scripts

### PR DESCRIPTION
**Pull request by jboyer4**
I lied - there were some changes to one of the scripts

### CODE REVIEW

**Comment by jboyer4** [comment]
Looks like I need to rebase on master first

**Comment by jboyer4** [comment]
Updated

File: src/ImageComparatorSgloh.hpp
Comment (F-Sossi) [inline src/ImageComparatorSgloh.hpp:1]: Can you do a pull from main before you push so we keep the fixes we put into the comaparators.

**Review by F-Sossi — CHANGES_REQUESTED** [review]
Can you do a pull from main before you push so we keep the fixes we put into the comaparators. The pthon looks goo you are just pushing the old version of the comaparators.

**Review by F-Sossi — APPROVED** [review]

**Pull request merged**